src/templates/App.js

```diff
@@ -25,6 +25,7 @@ export const App = (props) => {
             }
           `}
         />
+        <CookieNotice />
```

Comment:
This needs to be removed; it's already elsewhere in the page.

```suggestion
        <CookieNotice />
```

Reply:
Blarrrrrrrgh this is why I get you to review things, especially when I'm le tired.

Reply:
Fixed it in `master` 😆